tests/dragonfly/redis_replicaiton_test.py

```diff
@@ -5,13 +5,14 @@
 import subprocess
 from .utility import *
 
+
 class RedisServer:
     def __init__(self):
         self.port = 5555
         self.proc = None
 
     def start(self):
-        self.proc = subprocess.Popen(["redis-server-6.2.11",
+        self.proc = subprocess.Popen(["redis-server",
```

Comment:
This can be a problem - as it can be any version and we want to specifically use 6.2.11 which is the last redis version DF is compatible with.
It can be downloaded from here https://download.redis.io/releases/redis-6.2.11.tar.gz